manifest.json

```diff
@@ -134,5 +134,53 @@
       }
     ],
     "dependencies": { "language_tex": {}, "lsp": {} }
+  },{
+    "id": "quicklintjs",
```

Comment:
Might be a good idea to prefix this with `lsp_`.
Same thing for the plugin filename.

Reply:
i don't really see any need for that since other plugins are official languages LSP's, and do not operate as a linter only, and also since typescript server will be added soon that would be the default lsp for typescript/javascript. but however if you see its needed i will change it.

Reply:
I'll leave this to @adamharrison.

Reply:
already changed